### PR TITLE
added test cases for the methods get and count in file and db storage

### PR DESCRIPTION
**Pull request by luzperdomo92**
good morning, en those file added some test cases for the methods get and count in file storage and db_storage.

### CODE REVIEW

File: api/v1/app.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 
 @app.teardown_appcontext
-def teardown():
+def teardown(self):
```
Comment (luzperdomo92) [inline api/v1/app.py:20]: added self param for the method.

File: api/v1/views/__init__.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
 from flask import Blueprint
-from api.v1.views.index import *
 
 # Variable app_views which is an instance of Blueprint
 app_views = Blueprint('app_views', __name__, url_prefix='/api/v1')
+
+
+from api.v1.views.index import *
```
Comment (luzperdomo92) [inline api/v1/views/__init__.py:7]: in this case, the import must be on the bottom cause, the API never into the index.py

File: api/v1/views/index.py [hunk 3]
```diff
@@ -5,6 +5,6 @@
 
 
 @app_views.route('/status')
-def response():
+def status():
     """Jsonify the response and returns it"""
-    return jsonify("status": "OK")
+    return jsonify({"status": "OK"})
```
Comment (luzperdomo92) [inline api/v1/views/index.py:10]: add {} for dictionary syntax

File: api/v1/app.py (hunk 1 above)
Comment (EduardoVega04) [reply api/v1/app.py:20]: Kindergarden error from myself

**Pull request merged**